src/components/instantlaunches/InstantLaunchButtonWrapper.js

```diff
@@ -75,6 +80,12 @@ function InstantLaunchButtonWrapper(props) {
                 } else {
                     setOpen(false);
                 }
+                if (autolaunch) {
+                    // go to the analysis landing
+                    router.push(
+                        `/${NavigationConstants.ANALYSES}/${analysis?.id}`
+                    );
```

Comment:
The main question I have here is if I should use `router.replace` instead, so that the instant launch autolaunch page doesn't end up in the browser history (i.e., so that hitting "back" will go back to wherever someone clicked the badge from, not back to the launch page). Feeling like the answer is maybe yes - we do this for example to prevent hitting back and going back to the file create page.

Reply:
I convinced myself I should do this, heh, so I'm gonna go ahead and do that. No sense keeping any page that does something automatically in the browser history.